src/PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/Ricci.hpp

```diff
@@ -0,0 +1,132 @@
+ * \f}
+ * Gathering all terms with second derivatives:
+ * \f{align}
+ * R_{i j} =& \frac{1}{2} g^{k l} \left(\partial_k\Phi_{ijl}
+ *                                      + \partial_k\Phi_{jil}
+ *                                      - \partial_k\Phi_{lij}
+ *                                      + \partial_i\Phi_{lkj}
+ *                                      - \partial_i\Phi_{kjl}
+ *                                      - \partial_i\Phi_{jkl}\right)
+ *          + \mathcal{O}(\Phi), \nonumber\\
+ *         =& \frac{1}{2} g^{kl} \left(\partial_{(j}\Phi_{lki)}
+ *                                     - \partial_{(j}\Phi_{i)kl}
+ *                                     + \partial_k \Phi_{(ij)l}
+ *                                     - \partial_l \Phi_{kij} \right)
+ *          + \mathcal{O}(\Phi),
+ * \f}
+ * where we use the four-index constraint
+ * \f$C_{klij}=\partial_k\Phi_{lij}-\partial_l\Phi_{kij}=0\f$ to swap the
+ * first and second derivatives of the spatial metric, and symmetrize
+ * \f$R_{ij} = R_{(ij)}\f$. Similarly gathering the remaining terms and
+ * using the four-index constraint we get:
+ * \f{align}
+ * R_{i j} =& - b^k\left(\Phi_{ijk} + \Phi_{jik} - \Phi_{kij}\right)
+ *            -\frac{1}{2} \Phi_i{}^{kl} \left(\Phi_{jkl} + \Phi_{kjl}
+ *                                            - \Phi_{lkj}\right)\nonumber\\
+ *
+ *         &+ \frac{1}{2} d^k \left(\Phi_{ijk} + \Phi_{jik} - \Phi_{kij}\right)
+ *         - \left(\Phi_{(il)}{}^k - \frac{1}{2} \Phi^k{}_{il}\right)
+ *           \left(\Phi_{(kj)}{}^l - \frac{1}{2} \Phi^l{}_{kj}\right)
+ *         + \mathcal{O}(\partial\Phi) \\
+ *
+ *         =& \frac{1}{2} \left(\Phi_{ijk} + \Phi_{jik} - \Phi_{kij}\right)
+ *            (d^k - 2 b^k)
+ *         + \frac{1}{4} \Phi_{ik}{}^l \Phi_{jl}{}^k
+ *         + \frac{1}{2} \left(\Phi^k{}_{il} \Phi_{kj}{}^l
+ *                             - \Phi^k{}_{li} \Phi^l{}_{kj}\right)
+ *         + \mathcal{O}(\partial\Phi).
+ * \f}
+ * Gathering everyhing together, we compute the spatial Ricci tensor as:
+ * \f{eqnarray}\label{eq:rij}
+ * R_{i j} &=& \frac{1}{2} g^{kl} \left(\partial_{(j|}\Phi_{lk|i)}
+ *                                     - \partial_{(j}\Phi_{i)kl}
+ *                                     + \partial_k \Phi_{(ij)l}
+ *                                     - \partial_l \Phi_{kij}\right)\\
+ *         &+& \frac{1}{2} \left(\Phi_{ijk} + \Phi_{jik} - \Phi_{kij}\right)
+ *            (d^k - 2 b^k)
+ *          + \frac{1}{4} \Phi_{ik}{}^l \Phi_{jl}{}^k
+ *          + \frac{1}{2} \left(\Phi^k{}_{il} \Phi_{kj}{}^l
+ *                              - \Phi^k{}_{li} \Phi^l{}_{kj}\right).
+ * \f}
+ * This follows from equations (2.13) - (2.20) of \cite Kidder2001tz .
+ * Note that, in code, the mixed-index variables \f$\Phi_{ij}{}^k\f$ and
+ * \f$\Phi^i{}_{jk}\f$ are computed with a factor of \f$1/2\f$ and so
+ * the last 3 terms in Eq.(\f$\ref{eq:rij}\f$) that are quadratic in these
+ * terms occur multiplied by a factor of \f$4\f$.
+ */
```

Comment:
I don't think this last line is very clear here, because it references variables in the code that the reader hasn't looked at yet.  I think it's clearer is to rename the appropriate variables with `half` in front, and to put a comment in the code.

Reply:
Okay, I have rephrased this a bit, renamed appropriate variables with `half_` prepending their old names, and added comments in the cpp where these terms are being computed.